wicket-core/src/main/java/org/apache/wicket/markup/html/link/DownloadLink.java

```diff
@@ -65,6 +65,14 @@
 	 */
 	private Duration cacheDuration;
 
+	/**
+	 * Controls if the browser will save the file or display it inline.
+	 * <p>
+	 * The default is ATTACHMENT to maintain backward compatibility.
```

Comment:
I am not sure there is a need for this in the javadoc. If we want to say what is the default then maybe something like: `The default is ATTACHMENT because this is what what <em>download</em> implies.`

Reply:
I have no objection to that. 
Since I am not familiar with the github workflow: Must I adjust this text, or you will do it?